### PR TITLE
Added codesty.org

### PR DESCRIPTION
**Pull request by barbecue**
Community project engaged in creating competitive atmosphere for users and servers, chosen by its newest design & powerful structure.

### CODE REVIEW

File: websites/C/Codesty/presence.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+	} else if (pathName.startsWith('/server/')) {
+		presenceData.details = "Bir sunucu görüntülüyor:";
+		presenceData.state = document.title.replace('Codesty |', '').replace('Sunucusu', '');
+	} else if (pathName.startsWith('/u/')) {
+		presenceData.details = "Bir profil görüntülüyor:";
+		presenceData.state = document.title.replace('Codesty |', '').replace('Kullanıcısı', '');
+	} else {
+		presenceData.details = "Bir hata ile karşılaştı:";
+		presenceData.state = "404 Sayfa Bulunamadı";
+	}
```
Comment (SlowLife1661) [inline websites/C/Codesty/presence.ts:52]: I don't think this is good idea

File: websites/C/Codesty/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+   "$schema":"https://schemas.premid.app/metadata/1.0",
```
Comment (SlowLife1661) [inline websites/C/Codesty/dist/metadata.json:2]:
```suggestion
   "$schema":"https://schemas.premid.app/metadata/1.3",
```

File: websites/C/Codesty/presence.ts (hunk 1 above)
Comment (SlowLife1661) [reply websites/C/Codesty/presence.ts:52]: You already have this 
image

**Comment by ririxidev** [comment]
Heya @barbecue!
We are so sorry but we need to deny your pull request regarding this presence due of:
 - From what I see, site doesn't offer a lot for DOM readings etc.
 - Can't even access website to check it.

image

 
Regards,
**ririxi**. PreMiD Reviewer Team.
Contact: mail@ririxi.dev, ririxi#2721.

**Pull request closed without merging**